src/devices/cpu/dsp56156/dsp56156.cpp

```diff
@@ -117,7 +117,7 @@ void dsp56156_device::dsp56156_program_map(address_map &map)
 
 void dsp56156_device::dsp56156_x_data_map(address_map &map)
 {
-	map(0x0000, 0x07ff).ram();                              /* 1-5 */
+	//map(0x0000, 0x07ff).ram();                              /* 1-5 */
```

Comment:
Can you please add a comment on why this was commented out?  It’s useful for the next person reading the code to understand the purpose and why it was disabled.

Reply:
It should be possible to uncomment it now, I'll arrange a fix for it.

Reply:
Reinstated the handler, as it was previously commented out in order to allow the driver to capture all RAM accesses for logging purposes, which is no longer so necessary.